### DISCUSSION TITLE
Error with .setStatus

### DISCUSSION THREAD

**Discussion by IHaxU** [discordjs/discord.js#7775] [Q&A] [answered]
### Which package is this bug report for?

discord.js

### Issue description

Steps to reproduce:
1. Run the code sample
2. Error occurs
`TypeError: Cannot read properties of undefined (reading 'send')`

### Code sample

```typescript
const { Client, Intents } = require('discord.js');
const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MEMBERS, Intents.FLAGS.GUILD_PRESENCES, Intents.FLAGS.GUILD_MESSAGES] });

client.on('ready', () => {
  console.log(`Logged in as ${client.user.tag}!`);
  client.user.setStatus('dnd', 'test');
});

client.login("TOKEN");
```

### Package version

13.6.0

### Node.js version

v16.14.2

### Operating system

Ubuntu 20.04

### Priority this issue should have

Medium (should be fixed soon)

### Which partials do you have configured?

Not applicable (subpackage bug)

### Which gateway intents are you subscribing to?

Guilds, GuildMembers, GuildPresences, GuildMessages

**Comment by imranbarbhuiya** [comment, marked as answer]
https://discord.js.org/#/docs/discord.js/stable/class/ClientUser?scrollTo=setStatus


`setStatus` takes shardid as 2nd parameter and you r providing a string.
